Synapse/Api/Enum/ScpReconfType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Synapse.Api.Enum
+{
+    public enum ScpReconfType
```

Comment:
`ScpReconfType` -> `ScpRecontainmentType`
1. Typo
2. Don't be shy to type out stuff, "Recon" is a bit eh